### PR TITLE
feature/Autofill token with Password manager

### PR DESCRIPTION
**Pull request by daxter44**
#1322

Verified with 1Pass and Keepass

### CODE REVIEW

File: apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html [hunk 1]
```diff
@@ -8,12 +8,12 @@
   <div class="align-items-center d-flex flex-column">
     <mat-form-field appearance="outline" class="w-100">
       <mat-label i18n>Security Token</mat-label>
-      <textarea
+      <input
         cdkTextareaAutosize
```
Comment (dtslvr) [inline apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html:12]: Is this still working? If not, please remove.

File: apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html [hunk 2]
```diff
@@ -8,12 +8,12 @@
   <div class="align-items-center d-flex flex-column">
     <mat-form-field appearance="outline" class="w-100">
       <mat-label i18n>Security Token</mat-label>
-      <textarea
```
Comment (dtslvr) [inline apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html:11]: Can you remove the style in `login-with-access-token-dialog.scss`:

```
textarea.mat-input-element.cdk-textarea-autosize {
  box-sizing: content-box;
}
```

**Review by dtslvr — CHANGES_REQUESTED** [review]

File: apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html (hunk 1 above)
Comment (daxter44) [reply apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html]: Its not working with input field. I removed it

File: apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html (hunk 2 above)
Comment (daxter44) [reply apps/client/src/app/components/login-with-access-token-dialog/login-with-access-token-dialog.html:11]: I removed that.

**Comment by dtslvr** [comment]
I have just tested these changes. Is _Perform Autotype_ working for you? In _MacPass_ it does not fill the field and it clicks the checkbox 😞

**Comment by dtslvr** [comment]
There has been no activity on this issue for an extended period. Closing it now.

**Pull request closed without merging**